umm-spec-lib/src/cmr/umm_spec/umm_g/data_granule.clj

```diff
@@ -18,18 +18,35 @@
          :producer-gran-id (->> Identifiers
                                 (some #(when (= "ProducerGranuleId" (:IdentifierType %)) %))
                                 :Identifier)
+         :crid-ids (->> Identifiers
+                        (filter #(= "CRID" (:IdentifierType %)))
+                        (map :Identifier)
+                        seq) 
```

Comment:
I don't think we need to call `seq` here or for `:feature-ids` below.

Reply:
I think they are needed because otherwise it's possible that crid-ids would be [] when Identifiers don't have CRIDs. and we want crid-ids to be nil in that case. Same with feature-ids.

Reply:
You are right. I was thinking that a later step called `seq`, but there is no later step - this is the step that creates the model.